assemblyscript/mandelbrot.ts

```diff
@@ -25,27 +28,35 @@ function iterateEquation(x0: f64, y0: f64, maxiterations: i32): i32 {
   return iterations;
 }
 
-function scale(domainStart: f64, domainLength: f64, screenLength: f64, step: f64): f64 {
-  return domainStart + domainLength * ((step - screenLength) / screenLength);
+@inline
```

Comment:
When compiling this file with the TypeScript compiler it fails as follows:

~~~
error TS1206: Decorators are not valid here.
~~~

When I first looked at AssemblyScript I really liked the way you could compile the same codebase to either wasm or JS. I found this feature to be quite useful:

https://blog.scottlogic.com/2017/10/30/migrating-d3-force-layout-to-webassembly.html

Is this something you'd expect AssemblyScript to support